src/language-server/providers/schema/engine.ts

```diff
@@ -47,12 +50,13 @@ export class EngineSchemaProvider implements GraphQLSchemaProvider {
       );
     }
 
-    const { data, errors } = await this.client.execute<GetSchemaByTagQuery>({
+    const { data, errors } = await this.client.client.query({
```

Comment:
I can understand why you have the double `client.client` thing here, but this makes me wonder if you should add a public `query` function to `ApolloEngineClient` that just forwards to the internal client.

i.e.
```ts
public query<TData, TVariables>(options: QueryOptions<TVariables, TData>) { 
  return this.client.query(options);
}
```

Thoughts?